build/config/win/visual_studio_version.gni

```diff
@@ -22,7 +22,10 @@ if (is_starboard) {
   if (is_docker_build) {
     _default_visual_studio_path = "C:/BuildTools"
   } else {
-    _default_visual_studio_path = "C:/Program Files (x86)/Microsoft Visual Studio/2022/Professional"
+    _default_visual_studio_path = getenv("VS_INSTALL_DIR")
```

Comment:
should this be VSINSTALLDIR? I like this one, but we should probably convert other mentions of this if possible

Reply:
Done. Checked for more instances of VS_INSTALL_DIR and VS2022INSTALLDIR as well and we're all clear.